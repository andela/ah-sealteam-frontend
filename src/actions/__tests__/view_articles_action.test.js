import React from 'react';
import getAllArticles from '../view_articles.actions';
import configureStore from '../../store';

describe('<getAllArticles />', () => {
  it('renders  <getAllArticles /> components', () => {
    const snap = shallow(<getAllArticles store={configureStore()} slug="" />);
    expect(snap).toMatchSnapshot();
  });
});
