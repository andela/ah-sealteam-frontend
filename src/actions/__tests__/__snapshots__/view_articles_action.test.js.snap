// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<getAllArticles /> renders  <getAllArticles /> components 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <getAllArticles
    slug=""
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "slug": "",
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
    },
    "ref": null,
    "rendered": null,
    "type": "getAllArticles",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "slug": "",
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
      },
      "ref": null,
      "rendered": null,
      "type": "getAllArticles",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
